pkgs/applications/networking/bee/0001-nix-diff-for-bee-clef.patch

```diff
@@ -0,0 +1,41 @@
+From b9e5e9787f7f7a357fd872d7ce734900383d6404 Mon Sep 17 00:00:00 2001
+From: Attila Lendvai <attila@lendvai.name>
+Date: Sat, 30 Jan 2021 15:24:57 +0100
+Subject: [PATCH] nix diff for bee-clef
+
+---
+ packaging/bee-clef-service | 2 +-
+ packaging/bee-clef.service | 4 ++--
```

Comment:
Here you say you're removing the .service files. https://github.com/NixOS/nixpkgs/issues/113697#issuecomment-790121848 what's going on?

Reply:
i failed to clean up properly.